tsconfig.json

```diff
@@ -3,6 +3,12 @@
   "compilerOptions": {
     "baseUrl": "./",
     "paths": {
+      "@defichain-app-lib/actuator": [
+        "apps/libs/actuator/src"
+      ],
+      "@defichain-app-lib/actuator/*": [
+        "apps/libs/actuator/src/*"
+      ],
```

Comment:
```suggestion
      "@defichain-app-lib/*": [
        "apps/libs/*/src"
      ],
```

Reply:
![image](https://user-images.githubusercontent.com/31790206/154892638-18cfa47d-be39-47b9-a6d9-55431afa7e99.png)

Because jest is run at the project root, it works with `tsconfig.json` to figure out where this `@defichain-app-lib/actuator` actually is. Hence I don't think removing it here is an option.